test/gtest/plugin_manager.cpp

```diff
@@ -70,6 +70,60 @@ class LoadMultiplePluginsTestFixture
   }
 };
 
+class LoadedPluginTestFixture : public testing::Test {
+protected:
+  /* Plugin Manager. */
+  nixlPluginManager &plugin_manager_ = nixlPluginManager::getInstance();
```

Comment:
Hmm, having a designated naming convention for private/protected class fields (like '_' suffix here) is a popular convention in C++ projects, but it doesn't seem like other classes in NIXL adhere to it. Do we want to add those now or wait for converging on a code guidelines first?